roles/generate-jenkins/templates/README_SNIPPETS/KASM.j2

```diff
@@ -0,0 +1,127 @@
+**Modern GUI desktop apps have issues with the latest Docker and syscall compatibility, you can use Docker with the `--security-opt seccomp=unconfined` setting to allow these syscalls on hosts with older Kernels or libseccomp**
+
+### Security
+
+**Do not put this on the Internet if you do not know what you are doing**
+
+By default this container has no authentication and the optional environment variables `CUSTOM_USER` and `PASSWORD` to enable basic http auth via the embedded NGINX server should only be used to locally secure the container from unwanted access on a local network. If exposing this to the Internet we recommend putting it behind a reverse proxy, such as [SWAG](https://github.com/linuxserver/docker-swag), and ensuring a secure authentication solution is in place.
+
```

Comment:
I think we should mention why exposing it is a bad idea, and the direct consequences. 

```suggestion

As this container has easy shell access, it makes it a prime target for automated attacks for i.e. cryptominers, but could also be used to further probe your local network,

```

Reply:
I went with `From the web interface a terminal can be launched and it is configured for passwordless sudo, so anyone with access to it can install and run whatever they want along with probing your local network.`  let me know if this is sufficient.